src/main/java/world/bentobox/bentobox/lists/Flags.java

```diff
@@ -653,16 +653,21 @@ private Flags() {}
      * Controls who gets to harvest any crop related contents. e.g. Wheat, Sugar Cane, melon blocks, not stems, pumpkin blocks, etc.
      * @since 1.23.0
      */
-    public static final Flag HARVEST = new Flag.Builder("HARVEST", Material.PUMPKIN).type(Type.PROTECTION).build();
+    public static final Flag HARVEST = new Flag.Builder("HARVEST", Material.PUMPKIN).mode(Flag.Mode.BASIC).type(Type.PROTECTION).build();
```

Comment:
This was defaulting to advanced, but it's a basic ranked feature to help players share farming.